JDStatusBarNotification/JDStatusBarStyle.h

```diff
@@ -55,6 +60,9 @@ typedef NS_ENUM(NSInteger, JDStatusBarProgressBarPosition) {
 /// The animation, that is used to present the notification
 @property (nonatomic, assign) JDStatusBarAnimationType animationType;
 
+/// The duration of the bar animation
+@property (nonatomic) JDStatusBarAnimationDuration animationDuration;
```

Comment:
Just make this an NSTimeInterval.. remove all the whitespace changes, remove all enum related stuff.